nixos/modules/services/web-apps/limesurvey.nix

```diff
@@ -27,6 +27,7 @@ let
           ]
         ]
       );
+      return json_decode('${builtins.toJSON cfg.config}', true);
```

Comment:
That looks wrong

Reply:
Yeah the diff was messed up, thanks for catching it